src/test/resources/pages/instructorCourseEnrollError.html

```diff
@@ -37,7 +37,7 @@
 Team 3 | Frank Hughe
 Team 1 | Black Jack | bjack.gmail.tmt | This student email is invalid
 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa | Robert Downey | rob@email.tmt | This student team name is too long
-Team 2 | {*} | longname@email.tmt | This student name is too long
+Team 2 | {***} | longname@email.tmt | This student name is too long
```

Comment:
I suppose this is someone's name which doesn't contain space and is 62 characters long or more. Erroneously replaced?